styleguide/README.md

```diff
@@ -758,6 +758,10 @@ It's easier to read `expect(account).to be_valid` than `expect(subject).to be_va
 You're free to use `subject(:account) { … }`, but don't refer to it by the name `subject` elsewhere.
 
 If you do need it for technical reasons, perhaps with shared examples, go ahead.
+[RSpec documentation](https://relishapp.com/rspec/rspec-core/v/3-9/docs/subject/explicit-subject) says:
```

Comment:
I'd make this a new paragraph (so blank line above).

Maybe we should also remove line 758 since I think we don't ever use it, and it seems to run counter to RSpec recommendations.

For any changes like this that don't actually add new rules (where we want to check that we have consensus), feel free to just make the change and we can discuss post-merge in review :)

Reply:
758 _is_ the RSpec recommendation for `subject` as a regular `let`, but with "another intent-revealing name". My paraphrase.

Reply:
Hm, you're saying that RSpec recommends using `subject(:foo)` rather than `let(:foo)` for the subject of a test because it reveals that this isn't any old value? Interesting if so!

Reply:
Yes, just an annotation of a "special let". It can be scripted on, with the `subject` name, in all kinds of "programmatic" ways, but in one's expect() calls etc, it should use its named moniker (`foo`, in your example).